pkg/subctl/cmd/deploybroker.go

```diff
@@ -56,7 +56,7 @@ var validComponents = []string{components.ServiceDiscovery, components.Connectiv
 func init() {
 	deployBroker.PersistentFlags().BoolVar(&globalnetEnable, "globalnet", false,
 		"enable support for Overlapping CIDRs in connecting clusters (default disabled)")
-	deployBroker.PersistentFlags().StringVar(&globalnetCIDRRange, "globalnet-cidr-range", "169.254.0.0/16",
+	deployBroker.PersistentFlags().StringVar(&globalnetCIDRRange, "globalnet-cidr-range", "242.254.0.0/16",
```

Comment:
```suggestion
	deployBroker.PersistentFlags().StringVar(&globalnetCIDRRange, "globalnet-cidr-range", "242.0.0.0/8",
```

We used "/16" with LLA as it was the only range and we could not go beyond that. But while moving to 242.x.y.z, we can use the complete "/8", i.e., 242.0.0.0/8 which allows 16777214 addresses :)

Reply:
@sridhargaddam Then should we also change the CIDR size available to each cluster to a bigger number? 255 may not be enough, right?